### PR TITLE
Korean localization

### PR DESCRIPTION
**Pull request by aprilwebster**

### CODE REVIEW

File: i18n/es.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
 	"lang-spanish": "Español",
 	"lang-arabic": "Árabe",
 	"lang-japanese": "Japonés",
+	"lang-ko": "coreano",
```
Comment (germanattanasio) [inline i18n/es.json]: Coreano

File: i18n/es.json [hunk 2]
```diff
@@ -121,5 +122,6 @@
 	"twitterlang-es": "Español",
 	"twitterlang-ar": "Árabe",
 	"twitterlang-ja": "Japonés",
+	"twitterlang-ko": "coreano",
```
Comment (germanattanasio) [inline i18n/es.json]: Coreano

File: package.json [hunk 3]
```diff
@@ -1,15 +1,11 @@
 {
-  "name": "personality-insights-nodejs",
+  "name": "personality-insights-livedemo",
```
Comment (germanattanasio) [inline package.json]: nodejs.

File: i18n/es.json (hunk 1 above)
Comment (aprilwebster) [reply i18n/es.json]: Fixed.

File: i18n/es.json (hunk 2 above)
Comment (aprilwebster) [reply i18n/es.json]: Fixed.

File: package.json (hunk 3 above)
Comment (aprilwebster) [reply package.json]: Done.

**Pull request merged**